diff --git a/feeds/luci/applications/luci-app-mwan3helper/root/etc/init.d/mwan3helper b/feeds/luci/applications/luci-app-mwan3helper/root/etc/init.d/mwan3helper
index 6247ec2308..d846d2828e 100755
--- a/feeds/luci/applications/luci-app-mwan3helper/root/etc/init.d/mwan3helper
+++ b/feeds/luci/applications/luci-app-mwan3helper/root/etc/init.d/mwan3helper
@@ -148,21 +148,11 @@ start(){
   en=$(uci get mwan3helper.config.enabled 2>/dev/null)
   if [ $en -eq 1 ];  then
     start_pdnsd
-    mkdir -p /tmp/etc/dnsmasq-mwan3.d
-    ipset -N gfwlist hash:net 2>/dev/null
-    awk '!/^$/&&!/^#/{printf("ipset=/.%s/'"gfwlist"'\n",$0)}' /etc/mwan3helper/gfw.txt > /tmp/etc/dnsmasq-mwan3.d/mwan3_gfwlist.conf
-    awk '!/^$/&&!/^#/{printf("server=/.%s/'"127.0.0.1#53000"'\n",$0)}' /etc/mwan3helper/gfw.txt >> /tmp/etc/dnsmasq-mwan3.d/mwan3_gfwlist.conf
-    cat > /tmp/dnsmasq.d/dnsmasq-mwan3.conf <<EOF
-conf-dir=/var/etc/dnsmasq-mwan3.d
-EOF
-    /etc/init.d/dnsmasq restart
   fi
 }
 
 stop(){
 	  stop_pdnsd
-	  rm -f /tmp/dnsmasq.d/dnsmasq-mwan3.conf
-    /etc/init.d/dnsmasq restart
 }
 
 
