diff --git a/feeds/packages/utils/qemu/Makefile b/feeds/packages/utils/qemu/Makefile
index c832174..a706993 100644
--- a/feeds/packages/utils/qemu/Makefile
+++ b/feeds/packages/utils/qemu/Makefile
@@ -64,8 +64,6 @@ define Package/qemu-ga/install
 	$(INSTALL_BIN) ./files/qemu-ga.init $(1)/etc/init.d/qemu-ga
 	$(INSTALL_DIR) $(1)/etc/hotplug.d/virtio-ports
 	$(INSTALL_BIN) ./files/10-qemu-ga.hotplug $(1)/etc/hotplug.d/virtio-ports/10-qemu-ga
-	$(INSTALL_DIR) $(1)/sbin
-	$(INSTALL_BIN) ./files/shutdown $(1)/sbin/shutdown
 endef
 
 PKG_CONFIG_DEPENDS += CONFIG_PACKAGE_qemu-ga
diff --git a/feeds/packages/utils/qemu/files/shutdown b/feeds/packages/utils/qemu/files/shutdown
deleted file mode 100755
index 77d4751..0000000
--- a/feeds/packages/utils/qemu/files/shutdown
+++ /dev/null
@@ -1,2 +0,0 @@
-#!/bin/sh
-busybox poweroff
\ No newline at end of file
